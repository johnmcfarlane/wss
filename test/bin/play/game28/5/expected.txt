 65  3,3  | DYVOURS
 43  4,10 - DRYS
 42  3,10 - DORY
 42  4,10 - DRY
 41  3,10 - ROSY
 41  3,10 - SORY
 40  4,10 - ORY
 38  5,10 - SYUD
 36  5,10 - SYD
 35  5,10 - SYR
 35  6,10 - YOURS
 33  6,10 - YOUD
 32  6,10 - YODS
 32  6,10 - YOUR
 32  6,10 - YOUS
 31  6,10 - YOD
 31  6,10 - YOU
 31  6,10 - YUS
 30  3,10 - DORS
 30  6,10 - YD
 30  6,10 - YOR
 30  6,10 - YRS
 29  6,10 - SOURY
 29  6,10 - YO
 28  4,10 - ORS
 28  8,7  | SHROVY
 26  8,7  | SHROUDY
 26  6,10 - SOURD
 25  9,8  | SURV
 25  2,10 - YODS
 24  1,10 - DUROS
 24  5,1  | DYVOUR
 24  7,8  | GYROUS
 24  7,8  | GYRUS
 24  1,10 - SUDOR
 24  6,10 - SUDOR
 24  6,10 - SURV
 23  2,10 - OUDS
 23  6,10 - SYUD
 23  2,10 - URDS
 22  2,10 - DUOS
 22  9,4  | DUROS
 22  7,8  | GYROS
 22  8,7  | SHROUDS
 22  6,10 - SODY
 22  8,4  | SOUSHY
 22  3,9  | SOV
 22  6,10 - SOV
 22  1,10 - SYUD
 22  2,10 - UDOS
 22  2,10 - VOD
 22  1,10 - YOUD
 21  9,5  | DUOS
 21  7,8  | GYRO
 21  4,9  | PROV
 21  2,10 - RODS
 21  9,8  | SODY
 21  6,10 - SORY
 21  6,10 - SOUD
 21  6,10 - SURD
 21  6,10 - SYD
 21  9,8  | SYUD
 21  9,5  | UDOS
 21  3,5  | YOURS
 20  8,7  | SHROUD
 20  6,10 - SORD
 20  9,8  | SORY
 20  6,10 - SOUR
 20  9,8  | SOURY
 20  3,9  | SOURY
 20  6,10 - SOY
 20  6,10 - SUD
 20  3,9  | SURV
 20  6,10 - SYR
 20  3,9  | SYUD
 19  3,6  | DRUSY
 19  3,5  | DUROS
 19  6,6  | FROWY
 19  9,6  | ROSY
 19  6,10 - SOD
 19  3,9  | SODY
 19  6,10 - SOU
 19  9,8  | SOUD
 19  9,8  | SOURD
 19  3,9  | SOURD
 19  9,8  | SUDOR
 19  3,9  | SUDOR
 19  6,10 - SUR
 19  9,8  | SURD
 18  1,10 - ROUD
 18  9,8  | SORD
 18  3,9  | SOUD
 18  1,10 - SOUD
 18  9,8  | SOV
 18  3,9  | SOY
 18  1,10 - SURD
 18  3,9  | SYD
 18  2,10 - SYD
 18  8,6  - URDY
 18  2,10 - YOD
 17  9,6  | DOS
 17  3,6  | DRYS
 17  1,10 - DURO
 17  6,6  - FOURS
 17  6,10 - SO
 17  3,9  | SORY
 17  9,8  | SOUR
 17  3,9  | SOUR
 17  3,9  | SUD
 17  3,9  | SURD
 17  9,8  | SYD
 17  8,9  - URDY
 17  3,6  | YODS
 17  8,9  - YOUD
 17  3,6  | YOUS
 16  3,6  | DUOS
 16  3,7  | DYS
 16  6,6  | FROWSY
 16  2,10 - OUD
 16  3,6  | OUDS
 16  4,9  | PRUDY
 16  4,9  | PURDY
 16  9,6  | ROS
 16  3,7  | ROSY
 16  2,10 - RUD
 16  8,9  - RUDY
 16  3,9  | SOD
 16  3,9  | SORD
 16  1,10 - SORD
 16  3,9  | SOU
 16  9,8  | SOY
 16  9,8  | SUD
 16  2,10 - SUD
 16  9,8  | SYR
 16  3,9  | SYR
 16  3,6  | UDOS
 16  3,10 - UDS
 16  2,10 - URD
 16  3,6  | URDS
 16  8,6  - URDS
 16  8,9  - YOURS
 16  3,7  | YUS
 15  3,6  | DORS
 15  7,3  | DRYS
 15  2,10 - DUO
 15  6,6  - FOUD
 15  3,10 - ODS
 15  9,7  | OS
 15  3,6  | OURS
 15  4,9  | PORY
 15  4,9  | POSY
 15  4,9  | PYRUS
 15  3,6  | RODS
 15  3,6  | ROUS
 15  9,8  | SOD
 15  9,8  | SOU
 15  9,8  | SUR
 15  3,9  | SUR
 15  2,10 - UDO
 15  3,7  | UDS
 15  8,6  - URD
 15  8,9  - YODS
 15  3,10 - YOR
 15  5,3  | YOUR
 15  8,9  - YOUR
 15  8,9  - YOUS
 15  3,7  | YRS
 14  3,10 - DOR
 14  3,7  | DOS
 14  3,10 - DOS
 14  3,8  | DSR
 14  7,4  | DYS
 14  6,6  - FOUR
 14  3,7  | ODS
 14  2,10 - ORD
 14  7,3  | ORDU
 14  4,9  | PROUD
 14  4,9  | PUDSY
 14  2,10 - ROD
 14  8,9  - ROSY
 14  3,7  | RUS
 14  2,10 - SOD
 14  5,2  | SUDOR
 14  8,7  - SURV
 14  8,6  - UDOS
 14  3,7  | URS
 14  7,3  | YODS
 14  7,2  | YOURS
 14  7,3  | YOUS
 13  5,3  | DOUR
 13  7,3  | DROU
 13  5,3  | DURO
 13  7,2  | DUROS
 13  3,7  - DURR
 13  6,6  - FOU
 13  6,6  | FROWS
 13  8,9  - ORDU
 13  3,7  | ORS
 13  7,3  | OUDS
 13  4,9  | PORUS
 13  4,9  | POUSY
 13  4,9  | PRODS
 13  4,9  | PURSY
 13  3,7  | ROS
 13  3,10 - ROS
 13  8,9  - ROUD
 13  9,8  | SO
 13  7,6  | SOG
 13  8,6  - UDO
 13  8,9  - UDOS
 13  8,6  - UDS
 13  7,3  | URDS
 13  8,9  - URDS
 13  2,10 - URO
 13  3,8  | US
 13  8,9  - YOD
 13  7,4  | YOU
 13  8,9  - YOU
 13  7,4  | YUS
 13  8,9  - YUS
 12  3,7  - DORR
 12  7,5  | DU
 12  7,3  | DUOS
 12  4,10 - OR
 12  3,8  | OS
 12  4,10 - OS
 12  8,9  - OUDS
 12  4,9  | PROD
 12  4,9  | PROSY
 12  7,3  | RODS
 12  7,3  | ROUS
 12  3,9  | SO
 12  7,3  | UDOS
 12  7,4  | UDS
 12  8,6  - URO
 12  8,6  - URS
 12  8,9  - YD
 12  8,9  - YOR
 12  8,9  - YRS
 11  7,3  | DORS
 11  4,7  - DSR
 11  7,4  | ODS
 11  7,3  | OURS
 11  8,9  - OURS
 11  5,7  | OYERS
 11  4,9  | POURS
 11  4,9  | PRYS
 11  4,9  | PUDS
 11  4,9  | PYRO
 11  8,9  - RODS
 11  8,9  - ROUS
 11  7,4  | RUS
 11  7,4  | SOU
 11  5,3  | SOUR
 11  3,6  - SURFY
 11  8,9  - UDO
 11  8,9  - UDS
 11  8,9  - URD
 11  3,10 - YD
 11  8,9  - YO
 11  5,8  - YOGHS
 11  7,4  | YRS
 10  7,4  | DOS
 10  5,4  | DUO
 10  5,4  | DUR
 10  9,4  | DURO
 10  8,9  - ORY
 10  8,9  - OUD
 10  5,7  | OYER
 10  5,7  | OYES
 10  4,9  | PODS
 10  4,9  | POUR
 10  4,9  | PURS
 10  8,9  - ROY
 10  8,9  - RUD
 10  8,5  | RUSHY
 10  8,7  | SHOYU
 10  8,7  - SODY
 10  3,9  - SPEW
 10  8,7  - SUDSY
 10  5,4  | SYR
 10  8,7  - SYRUS
 10  8,7  - SYUD
 10  5,4  | UDO
 10  8,9  - URO
 10  7,4  | URS
 10  8,9  - URS
 10  7,5  | US
 10  5,4  | YOR
  9  5,4  | DOR
  9  5,4  | DSR
  9  9,5  | DUO
  9  3,10 - OD
  9  8,9  - ODS
  9  8,9  - ORD
  9  7,4  | ORS
  9  5,4  | OUR
  9  8,9  - OUR
  9  8,9  - OY
  9  4,9  | PROS
  9  4,9  | PUY
  9  8,9  - ROD
  9  7,4  | ROS
  9  8,9  - RUS
  9  8,7  - SORY
  9  8,7  - SOURY
  9  8,7  - SUDORS
  9  5,4  | SUR
  9  8,7  - SURDS
  9  9,5  | UDO
  9  8,6  - UR
  9  8,9  - UR
  9  5,4  | URO
  9  8,6  - US
  9  8,9  - US
  9  5,5  | YO
  9  3,10 - YO
  8  5,5  | DO
  8  3,10 - DO
  8  3,6  - DROF
  8  8,5  | DUSH
  8  8,9  - OD
  8  8,9  - ORS
  8  7,5  | OS
  8  4,9  | POY
  8  4,9  | PRY
  8  4,9  | PUD
  8  4,9  | PYR
  8  8,9  - ROS
  8  8,7  - SORDS
  8  8,7  - SORUS
  8  8,7  - SOUD
  8  8,7  - SOURD
  8  8,7  - SUDOR
  8  8,7  - SURD
  8  3,6  - SURF
  8  5,5  | UR
  8  9,5  | URO
  7  5,5  | OR
  7  8,9  - OR
  7  8,9  - OS
  7  4,6  - OUF
  7  4,9  | POD
  7  4,9  | PUR
  7  4,9  | PUS
  7  8,5  | RUSH
  7  8,7  | SHOD
  7  8,7  | SHOU
  7  8,7  | SHY
  7  5,5  | SO
  7  3,10 - SO
  7  8,7  - SORD
  7  8,7  - SOURS
  7  8,7  - SOV
  7  8,7  - SOYS
  7  8,7  - SUDS
  7  4,6  - SUF
  7  5,6  - UFO
  7  5,6  - UFS
  7  4,6  - URF
  7  5,8  | YERD
  6  6,6  - FU
  6  4,6  - ORF
  6  4,9  | PD
  6  4,9  | POR
  6  4,9  | POS
  6  4,9  | PRO
  6  4,9  | PRS
  6  5,6  - RFS
  6  8,7  | SHOR
  6  8,7  | SHU
  6  8,7  - SODS
  6  8,5  | SOSH
  6  8,7  - SOUR
  6  8,7  - SOUS
  6  8,7  - SYD
  6  8,6  | USH
  6  5,8  | YESO
  6  9,6  | YO
  5  9,6  | DO
  5  6,6  - FS
  5  5,6  - OF
  5  4,9  | PO
  5  5,6  - RF
  5  8,7  | SHO
  5  8,7  | SHR
  5  8,7  - SOY
  5  8,7  - SUD
  5  8,7  - SYR
  5  5,8  | YER
  5  5,8  | YES
  4  8,7  - SDS
  4  9,6  | SO
  4  8,7  - SOD
  4  8,7  - SOU
  4  8,7  - SSU
  4  8,7  - SUR
  4  8,7  - SUS
  3  8,7  - SOS
  2  8,7  - SO
