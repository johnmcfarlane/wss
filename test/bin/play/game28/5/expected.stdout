 65  3,3  | DYVOURS
 24  1,10 - DUROS
 24  6,10 - SUDOR
 22  2,10 - DUOS
 22  9,4  | DUROS
 22  8,7  | SHOYUS
 22  8,7  | SHROUDS
 22  2,10 - UDOS
 21  9,5  | DUOS
 21  6,10 - SURD
 21  9,5  | UDOS
 21  3,5  | YOURS
 20  8,7  | SHROUD
 20  6,10 - SORD
 20  6,10 - SOUR
 20  6,10 - SOY
 19  3,5  | DUROS
 19  9,6  | ROSY
 19  6,10 - SOD
 19  6,10 - SOU
 19  9,8  | SUDOR
 19  3,9  | SUDOR
 19  9,8  | SURD
 18  9,8  | SORD
 18  3,9  | SOY
 18  7,5  | VU
 17  9,6  | DOS
 17  3,6  | DRYS
 17  1,10 - DURO
 17  6,10 - SO
 17  9,8  | SOUR
 17  3,9  | SOUR
 17  3,9  | SURD
 17  3,6  | YODS
 17  8,9  - YOUD
 17  3,6  | YOUS
 16  3,6  | DUOS
 16  6,6  | FROWSY
 16  3,6  | OUDS
 16  3,7  | ROSY
 16  3,9  | SOD
 16  3,9  | SORD
 16  3,9  | SOU
 16  9,8  | SOY
 16  3,6  | UDOS
 16  3,6  | URDS
 16  8,9  - YOURS
 15  3,6  | DORS
 15  2,10 - DUO
 15  9,7  | OS
 15  3,6  | OURS
 15  4,9  | POSY
 15  3,6  | RODS
 15  9,8  | SOD
 15  9,8  | SOU
 15  2,10 - UDO
 15  8,9  - YODS
 15  8,9  - YOUR
 15  8,9  - YOUS
 14  3,7  | DOS
 14  3,10 - DOS
 14  3,7  | ODS
 14  4,9  | PROUD
 13  5,3  | DURO
 13  3,7  - DURR
 13  6,6  | FROWS
 13  3,7  | ORS
 13  4,9  | PRODS
 13  4,9  | PURSY
 13  9,8  | SO
 13  3,8  | US
 13  8,9  - YOD
 13  7,4  | YOU
 13  8,9  - YOU
 12  3,7  - DORR
 12  3,8  | OS
 12  4,10 - OS
 12  4,9  | PROD
 12  4,9  | PROSY
 12  4,9  | PYROS
 12  3,9  | SO
 11  5,7  | OYERS
 11  4,9  | POURS
 11  4,9  | PUDS
 11  4,9  | PYRO
 11  7,4  | SOU
 11  3,6  - SURFY
 11  8,9  - YO
 11  5,8  - YOGHS
 10  5,4  | DUO
 10  9,4  | DURO
 10  5,7  | OYER
 10  5,7  | OYES
 10  4,9  | PODS
 10  4,9  | POUR
 10  4,9  | PURS
 10  8,5  | RUSHY
 10  8,7  | SHOYU
 10  3,9  - SPEW
 10  8,7  - SUDSY
 10  5,4  | UDO
  9  9,5  | DUO
  9  4,9  | PROS
  9  8,7  - SUDORS
  9  8,7  - SURDS
  9  9,5  | UDO
  9  5,5  | YO
  9  3,10 - YO
  8  5,5  | DO
  8  3,10 - DO
  8  4,9  | PRY
  8  4,9  | PUD
  8  8,7  - SORDS
  8  8,7  - SORUS
  8  8,7  - SUDOR
  8  8,7  - SURD
  8  3,6  - SURF
  7  4,9  | POD
  7  4,9  | PUR
  7  4,9  | PUS
  7  8,5  | RUSH
  7  8,7  | SHOD
  7  8,7  | SHY
  7  5,5  | SO
  7  3,10 - SO
  7  8,7  - SORD
  7  8,7  - SOURS
  7  8,7  - SOYS
  7  8,7  - SUDS
  6  6,6  - FU
  6  4,6  - ORF
  6  4,9  | POS
  6  4,9  | PRO
  6  8,7  - SODS
  6  8,7  - SOUR
  6  8,7  - SOUS
  6  9,6  | YO
  5  9,6  | DO
  5  5,6  - OF
  5  4,9  | PO
  5  8,7  - SOY
  5  5,8  | YES
  4  9,6  | SO
  4  8,7  - SOD
  4  8,7  - SOU
  3  8,7  - SOS
  2  8,7  - SO
